README.md

```diff
@@ -0,0 +1,36 @@
+# Content Server
+
+This is the server serving the in-game client for OpenTTD's content service, called BaNaNaS.
+It works together with [https://github.com/OpenTTD/content-api](content-api), which serves the HTTP API.
+
+## Development
+
+This API is written in Python 3.8 with aiohttp, and makes strong use of asyncio.
+
+### Running a local server
+
+To start it, you are advised to first create a virtualenv:
+
+```bash
+python3 -m venv .env
+.env/bin/pip install -r requirements.txt
```

Comment:
This is an unusual way of using a venv - you usually activate it (`source .env/bin/activate`) then just call `pip` directly

Reply:
This is pretty common tbh. And both methods are identical. This is slightly safer in READMEs, we found out, as people returning to their shell without an "activate" are still doing the right thing. Bit tomato tomato I guess :)